src/features/configTable/configTable.tsx

```diff
@@ -0,0 +1,222 @@
+    }
+
+    const nameSurnamePair = `${employee.name} ${employee.surName}`;
+    if (nameSurnamePairs[nameSurnamePair] && nameSurnamePair != " ") {
+      employeeErrors.name = "Name and surname pair must be unique";
+      employeeErrors.surName = "Name and surname pair must be unique";
+    } else {
+      nameSurnamePairs[nameSurnamePair] = true;
+    }
+
+    return employeeErrors;
+  });
+
+  return errors;
+};
+
+export default function EditableTableExample(props: {
+  title: string;
+  data: Employee[];
+}) {
+  return (
+    <Form
+      onSubmit={onSubmit}
+      initialValues={{ configData: props.data }}
+      mutators={{
+        ...arrayMutators,
+      }}
+      validate={validateForm}
+      render={({
+        handleSubmit,
+        form: {
+          mutators: { push, pop },
+        },
+        form,
+        hasValidationErrors,
+      }: FormRenderProps<{ configData: Employee[] }>) => {
+        const { pristine, submitting, errors } = form.getState();
+        return (
+          <VStack as="form" onSubmit={handleSubmit} spacing={6}>
+            <Heading>{props.title}</Heading>
+            <HStack spacing={4}>
+              <Button
+                rightIcon={<FaPlus />}
+                onClick={() =>
+                  push("configData", {
+                    name: "",
+                    surName: "",
+                    employed: false,
+                  })
+                }
+                colorScheme="green"
+                aria-label="Add Employee"
+              >
+                New Employee
+              </Button>
+              <Button
+                type="submit"
+                isDisabled={submitting || pristine || hasValidationErrors}
+              >
+                Submit
+              </Button>
+              <Button
+                onClick={() => form.reset()}
+                isDisabled={submitting || pristine}
+              >
+                Reset
+              </Button>
+            </HStack>
+            <Table variant="simple">
+              <Thead>
+                <Tr>
+                  <Th>First Name</Th>
+                  <Th>Last Name</Th>
+                  <Th>Employed</Th>
+                  <Th>Actions</Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                <FieldArray name="configData">
+                  {({ fields }) =>
+                    fields
+                      .map((employee, index) => (
+                        <Tr key={employee}>
+                          <Td>
+                            <Field
+                              name={`${employee}.name`}
+                              render={({ input, meta }) => (
+                                <Box>
+                                  <FormControl>
+                                    <Input
+                                      {...input}
+                                      placeholder="First Name"
+                                      isInvalid={
```

Comment:
this 'is invalid' is repeaded too many times. also if I remember the library, it should be taken from meta.

Please add Chakra control helpers as I did which factor this out.

Reply:
isInvalid is the standard for Chackra.
The control helper <FormErrorMessage> is not working because it should be used in combination with "meta" that we do not use since we have per row validation.
<FormHelperText> is still a Chackra control helper originally not designed to be an error but it differs only for the color.